### PR TITLE
Adding loader for SCMS

### PR DESCRIPTION
**Pull request by genisplaja**
### Description

Please include the following information at the top level docstring for the dataset's module mydataset.py:

- [x] Describe annotations included in the dataset
- [x] Indicate the size of the datasets (e.g. number files and duration, hours)
- [x] Mention the origin of the dataset (e.g. creator, institution)
- [x] Describe the type of music included in the dataset
- [x] Indicate any relevant papers related to the dataset

#### Dataset loaders checklist:

- [x] Create a script in `scripts/`, e.g. `make_my_dataset_index.py`, which generates an index file.
- [x] Run the script on the canonical version of the dataset and save the index in `mirdata/indexes/` e.g. `my_dataset_index.json`.
- [x] Create a module in mirdata, e.g. `mirdata/my_dataset.py`
- [x] Create tests for your loader in `tests/datasets/`, e.g. `test_my_dataset.py`
- [x] Add your module to `docs/source/mirdata.rst` and `docs/source/table.rst`
- [x] Run `tests/test_full_dataset.py` on your dataset.

### CODE REVIEW

File: mirdata/datasets/scms.py [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+                "title": " ".join(idx[0].split("_")[:-1]),
+                "tonic": artist_info[idx[1]]["tonic"],
+                "gender": artist_info[idx[1]]["gender"],
+                "train": artist_info[idx[1]]["train"],
+            }
+
+        return metadata
+
+    def load_audio(self, *args, **kwargs):
+        return load_audio(*args, **kwargs)
```
Comment (magdalenafuentes) [inline mirdata/datasets/scms.py:296]: This line is not covered?

File: mirdata/datasets/scms.py [hunk 2]
```diff
@@ -0,0 +1,302 @@
...
+
+    def load_pitch(self, *args, **kwargs):
+        return load_pitch(*args, **kwargs)
```
Comment (magdalenafuentes) [inline mirdata/datasets/scms.py:299]: Same here, is this the issue we discussed today that appears in most loaders?

File: mirdata/datasets/scms.py [hunk 3]
```diff
@@ -0,0 +1,302 @@
...
+
+    def load_activations(self, *args, **kwargs):
+        return load_activations(*args, **kwargs)
```
Comment (magdalenafuentes) [inline mirdata/datasets/scms.py:302]: Same here

**Review by magdalenafuentes — CHANGES_REQUESTED** [review]
Hey @genisplaja, looks great, left just minor comments. Let me know your thoughts and we can merge soon!

File: docs/source/quick_reference.rst [hunk 4]
```diff
@@ -47,6 +47,15 @@ Here we provide a rough guide to the types in this table, but we **strongly reco
...
 
+.. _activations:
+
+Activations
+^^^^^^^^^^^
+A generic annotation to indicate whether a particular event is happening at a given time. 
+It can be used, for instance, to indicate whether a particular instrument is playing at a 
+given time-step or whether a particular note is being played at a given time-step. In fact,
+it is implicit in annotations such as F0_ or Vocal Notes_ (instrument is activated when the
+melody is non-0). However, some datasets provide it as a standalone annotation.
```
[the remark this answers is not in the document]
Comment (genisplaja) [reply docs/source/quick_reference.rst:58]: Right! I'm updating that :)

File: mirdata/datasets/scms.py [hunk 5]
```diff
@@ -0,0 +1,302 @@
...
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        artist (str): artist
+        audio_path (str): path to the audio file
+        pitch_path (str): path to the pitch annotation file
+        activations_path (str): path to the vocal activation annotation file
+        tonic (str): tonic of the recording
+        gender (str): genre
```
[the remark this answers is not in the document]
Comment (genisplaja) [reply mirdata/datasets/scms.py:80]: upss

File: mirdata/datasets/scms.py (hunk 2 above)
Comment (genisplaja) [reply mirdata/datasets/scms.py:299]: missed testing `load_audio` actually (done it now), but `load_pitch` and `load_activations` are actually tested, so I wonder if yeah, is one of these testing problems we discussed...

Comment (genisplaja) [reply mirdata/datasets/scms.py]: ok sorry @magdalenafuentes, these functions should not be there, were deprecated a few versions ago. The newest loaders do not have them actually, started from the wrong template hehe. Fixed!

**Review by magdalenafuentes — APPROVED** [review]
LGTM! Thanks @genisplaja

**Pull request merged**